package.json

```diff
@@ -8,13 +8,19 @@
   "license": "MIT",
   "type": "module",
   "devDependencies": {
+    "@vuepress/client": "2.0.0-beta.51",
     "remark-lint": "^9.1.1",
+    "vue": "^3.2.39",
     "vuepress": "2.0.0-beta.51"
   },
   "scripts": {
     "docs:dev": "vuepress dev docs",
     "docs:build": "vuepress build docs",
     "lint": "yarn remark docs/ --use remark-preset-lint-recommended -- use remark-preset-link-markdown-style-guide"
   },
-  "dependencies": {}
+  "dependencies": {
+    "fs": "0.0.1-security",
+    "path": "^0.12.7",
```

Comment:
Why don't we use Node's internal `fs` and `path` modules?

Reply:
This just needs to be deleted tbh. These were me trying to get a sidebar hack to work, and ended up on sidebar groups instead....which will have to be further refined as we go forward....

Reply:
```suggestion
```